### PR TITLE
feat: add diko vault rewards to UI

### PR DESCRIPTION
**Pull request by philipdesmedt**

### CODE REVIEW

File: clarity/contracts/arkadiko-freddie-v1-1.clar [hunk 1]
```diff
@@ -80,9 +80,9 @@
             (ok
               (/
                 (* (get collateral vault) (get last-price-in-cents stx-price-in-cents))
+                (get debt vault)
                 ;; TODO: cost is too high to use this in the front-end - fix this
-                ;; (+ (get debt vault) u0);; (unwrap-panic (get-stability-fee-for-vault vault-id)))
-                (+ (get debt vault) u0 (unwrap-panic (get-stability-fee-for-vault vault-id)))
+                ;; (+ (get debt vault) (unwrap-panic (get-stability-fee-for-vault vault-id)))
```
Comment (nieldeckx) [inline clarity/contracts/arkadiko-freddie-v1-1.clar:85]: Q: Any idea what's causing high cost?

File: web/components/manage-vault.tsx [hunk 2]
```diff
@@ -119,6 +120,17 @@ export const ManageVault = ({ match }) => {
...
+      const rewardCall = await callReadOnlyFunction({
+        contractAddress,
+        contractName: "arkadiko-vault-rewards-v1-1",
+        functionName: "get-pending-rewards",
+        functionArgs: [standardPrincipalCV(senderAddress || '')],
+        senderAddress: contractAddress || '',
+        network: network
+      });
+      const reward = cvToJSON(rewardCall);
+      setPendingVaultRewards(reward.value.value / 1000000);
```
Comment (nieldeckx) [inline web/components/manage-vault.tsx:133]: NTF: method to convert integers from smart contracts to doubles with given amount of decimals to show to the user.

**Review by nieldeckx — COMMENTED** [review]
Looking forward to the demo!

File: clarity/contracts/arkadiko-freddie-v1-1.clar (hunk 1 above)
Comment (philipdesmedt) [reply clarity/contracts/arkadiko-freddie-v1-1.clar:85]: Probably combination of the `pow` function in `get-stability-fee-for-vault` and the other contract calls we do in that function

File: web/components/manage-vault.tsx (hunk 2 above)
Comment (philipdesmedt) [reply web/components/manage-vault.tsx:133]: Good point. I'll do that in another PR!

Comment (philipdesmedt) [reply web/components/manage-vault.tsx:133]: @nieldeckx added card for this - https://trello.com/c/UmfZxZ3Z/104-simplify-front-end-logic

**Pull request merged**